formapp/app.py

```diff
@@ -41,6 +41,7 @@
 from werkzeug.utils import secure_filename
 from websocket_server import WebsocketServer
 from cryptography.fernet import Fernet
+from playsound import playsound
```

Comment:
We already use pygame for this. Look through the code in this file to find examples.